docs: add virtual hosts articles ADD and EDIT (#945)
devopness/devopness: docs/virtual-hosts/edit-virtual-host.md

---
title: Edit a Virtual host

> intro: Edit a Virtual host to change how your applications are accessible on the internet by IP address or domain name

Reviewer: ```suggestion
intro: Edit a Virtual Host to change how an application is accessible on the internet , by server IP address or a domain name
```
